### PR TITLE
New article: What did Santa forget?

### PR DESCRIPTION
**Pull request by bbrtj**
#226 

Not sure what to do with blocks presenting output, so I left them idented.

### CODE REVIEW

**Comment by thibaultduponchelle** [comment]
Preview:
forget

**Comment by thibaultduponchelle** [comment]
I would have added a line in the conclusion like: "Thanks to this, if you were nice, dear reader, you do not have any chance to be forgotten" (but better said)

**Comment by thibaultduponchelle** [comment]
The article is OK for me

**Review by thibaultduponchelle — APPROVED** [review]

**Comment by bbrtj** [comment]
Well I don't know, I don't really like obvious conclusions. But if you think the article ends weirdly or unexpectedly I'll try to come up with something.

**Comment by thibaultduponchelle** [comment]
> Well I don't know, I don't really like full-blown / obvious conclusions. But if you think the article ends weirdly or unexpectedly I'll try to come up with something.

No it's OK, was just a proposal, let's keep like this then

File: 2022/incoming/what-did-santa-forget-.pod [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+Topic: Diff two Perl structures
+
+=encoding utf8
+
+=head1 What did Santa forget?
+
+Santa is about ready to start his yearly trip around the world! But does he
+I<really> have everything?
+
+Would be silly not to deliver a present to someone who has been trying the
```
Comment (openstrike) [inline 2022/incoming/what-did-santa-forget-.pod:12]: Suggestion: s/Would/It would/

File: 2022/incoming/what-did-santa-forget-.pod [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+whole year to be nice. Luckily, to avoid such blunder, he made a list
```
Comment (openstrike) [inline 2022/incoming/what-did-santa-forget-.pod:13]: Suggestion: s/such blunder/such a blunder/

File: 2022/incoming/what-did-santa-forget-.pod [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+        'candy canes',
+        'fireplace break-in tool',
+        'cookies degustation kit',
+    ],
+);
+
+=end perl
+
+Now all he needs to do is compare it to the list of things he packed. He asked
+elves how should he do it, and they suggested giving a try to recently released
```
Comment (openstrike) [inline 2022/incoming/what-did-santa-forget-.pod:42]: Maybe "asked the elves"?

**Review by openstrike — CHANGES_REQUESTED** [review]
This looks in good shape. Just a few minor suggestions to make it read more naturally.

File: 2022/incoming/what-did-santa-forget-.pod (hunk 1 above)
Comment (bbrtj) [reply 2022/incoming/what-did-santa-forget-.pod]: ++

File: 2022/incoming/what-did-santa-forget-.pod (hunk 2 above)
Comment (bbrtj) [reply 2022/incoming/what-did-santa-forget-.pod]: ++

File: 2022/incoming/what-did-santa-forget-.pod (hunk 3 above)
Comment (bbrtj) [reply 2022/incoming/what-did-santa-forget-.pod:42]: ++

**Comment by bbrtj** [comment]
Thanks @openstrike, fixed.

**Review by openstrike — APPROVED** [review]
Thanks, all looks good now.

**Comment by thibaultduponchelle** [comment]
Thank you @openstrike 
Thank you @bbrtj 

I merge it now

**Pull request merged**